MozillaFoundation/foundation.mozilla.org
# Long author names now wrap instead of breaking out of container

# Description



Link to sample test page: https://foundation-s-10093-brea-y7a8gy.mofostaging.net/en/research/library/skill-look-pretty-article-along-himself-design/
Related PRs/issues: #10093


Per request of design, this PR updates the div that houses the author profile's name and tagline with the rule `break-all`, so profiles with extra long last names or bios wrap instead of breaking out of the containing div.


# Screenshots


**Before:**
<img width="350" src="https://images.zenhubusercontent.com/61e1e4f0d1fded7471402d12/62971a27-49d7-4462-99a0-021e596b384a">


**After:**

<img width="350" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/de5022c2-9075-4763-8f7a-d17208071506">

# Step to test

1. Visit the test page linked above
2. Scroll down to the authors list section
3. If the test name "Susan Shermanmanman" is wrapping as expected, testing is complete!
